src/controllers/activegate/reconciler/automaticapimonitoring/reconciler.go

```diff
@@ -9,14 +9,14 @@ import (
 
 type AutomaticApiMonitoringReconciler struct {
 	dtc            dtclient.Client
-	name           string
+	clusterLabel   string
```

Comment:
```suggestion
	clusterName   string
```

Reply:
https://github.com/Dynatrace/dynatrace-operator/pull/797/files#diff-c692bd7dc2ed9d2d66a9003766034a3a98d19d42be7267433386415031b17426R81 -> technically it's a label, we agreed on annotation with `name` but source code should reflect actual reality.